packages/sil-voting-app/src/routes/dashboard/components/Proposal/RenderedProposal.tsx

```diff
@@ -0,0 +1,24 @@
+import Hairline from 'medulas-react-components/lib/components/Hairline';
+import React from 'react';
+
+import Proposal, { ProposalProps } from '.';
+
+interface Props {
+  readonly proposal: ProposalProps;
+  readonly index: number;
+}
+
+const RenderedProposal = (props: Props): JSX.Element => {
+  if (props.index === 0) {
+    return <React.Fragment key={props.index}>{Proposal(props.proposal)}</React.Fragment>;
+  }
+
+  return (
+    <React.Fragment key={props.index}>
+      <Hairline />
+      {Proposal(props.proposal)}
```

Comment:
Why don't you want to use it as component? You can pass `props.proposal` as `<Proposal {...props.proposal} />`.

Reply:
Must be a remnant of some old code, I've just changed it.